Document how to use modems is various ways

Lupus590 opened the pull request:
#1068

Lupus590 committed:
maybe too many images

Lupus590 committed:
grammarly fixes

Lupus590 committed:
fix typo in code example

SquidDev reviewed (changes requested):
Thank you so much for putting this together. Looking at this, I really think it's worth splitting each section into separate pages - while three are all about modems, the underlying topics are pretty different.

 - Moving items and fluids: I really like what you've done here, especially with the concrete colours - definitely worth linking this page in the inventory and fluid_handler peripheral doc.

 - Wired modems as peripheral connectors: I'm not quite sure what to do with this section. There's _some_ stuff about wired networks already on the main [peripheral] docs - I'm not sure whether it's worth trying to flesh those out, or if it is worth the extra docs - not sure if people have thoughts on that?

[peripheral]: https://tweaked.cc/module/peripheral.html

Lupus590 committed:
Merge branch 'mc-1.16.x' of https://github.com/SquidDev-CC/CC-Tweaked into docs/using_modems

Lupus590 committed:
images -> /images

Lupus590 committed:
:::Note -> :::note

Lupus590 committed:
add a short bit on peripheral.find

Lupus590 committed:
Merge branch 'mc-1.16.x' of https://github.com/SquidDev-CC/CC-Tweaked into docs/using_modems

Lupus590 committed:
Split the file.

Lupus590 committed:
add links to the new docs on the generic peripheral pages

Lupus590 committed:
add fluid transportation stuff

I might copy the item one and do a find and replace
I'm not looking forward to doing new screenshots though.

Lupus590 committed:
Change header part

Lupus590 committed:
delete rednet bit

Lupus590 commented:
Right, I need to read through these again to check that they flow well and that I haven't deleted something that actually was important. Not going to do that now as I feel quite tired, a job interview at 8am when you're used to getting up at 2pm will do that.

SquidDev commented:
Hope the job interview went well!

Lupus590 committed:
Merge branch 'mc-1.16.x' of https://github.com/SquidDev-CC/CC-Tweaked into docs/using_modems

Lupus590 committed:
Merge branch 'mc-1.16.x' of https://github.com/SquidDev-CC/CC-Tweaked into docs/using_modems

Lupus590 committed:
Fix top level  headings

Lupus590 committed:
Move and rewrite fluids note

Lupus590 committed:
Spelling grammar stuff

Lupus590 committed:
Distracted changes

I want to finish the rednet page first

Lupus590 committed:
WIP rewrite

Lupus590 commented:
I might move the big bullet point section into its own guide, it's a bit in the way where it is and could still do with being referenced in the item movement page.

Lupus590 committed:
cleanup

Lupus590 committed:
Merge branch 'mc-1.16.x' of https://github.com/SquidDev-CC/CC-Tweaked into docs/using_modems

Lupus590 committed:
fix top headings again

Lupus590 committed:
add about_modems

Lupus590 commented:
`about_modems` is ready for review, everything else is being worked on still.

Lupus590 committed:
basic_rednet ready for review

Lupus590 committed:
file header stuff

Lupus590 committed:
so much for this page being ready for review

Lupus590 committed:
misc changes

Lupus590 committed:
add peripheral connection section

Lupus590 committed:
move footnote to the file that it should be in

Lupus590 commented:
> Hope the job interview went well!

Well, they offered me the job, so I assume it went well.

SquidDev commented:
Oh wow. Congratulations!

Lupus590 committed:
Merge branch 'mc-1.20.x' into docs/using_modems
